examples/basic.ts

```diff
@@ -72,6 +72,7 @@ function makeControls(amount: number): IControlData[] {
 client.open({
     authToken: process.argv[2],
     versionId: parseInt(process.argv[3], 10),
+    url: process.argv[4] || null,
```

Comment:
I'd rather not push 'weirdness' to a public example for functionality that's only useful for internal testing.